src/picongpu/include/particles/Particles.tpp

```diff
@@ -174,10 +174,18 @@ void Particles<T_ParticleDescription>::update(uint32_t )
         InterpolationScheme,
         fieldSolver::NumericalCellType > FrameSolver;
 
+    // adjust interpolation area in particle pusher to allow sub-sampling pushes
+    typedef typename PMacc::math::CT::add< LowerMargin,
+    typename PMacc::math::CT::make_Int<LowerMargin::dim, picongpu::UsedParticlePusher::extendMargin>::type
```

Comment:
*offline discussion with @psychocoderHPC:*

This approach is not  possible because the pusher has no access to `FrameType`.
This a separate treatment of pusher extension and interpolation margin is needed.